src/lib/firebase.ts

```
@@ -3,37 +3,73 @@ import { getAuth, GoogleAuthProvider, Auth } from "firebase/auth";
+    '- NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID\n' +
+    '- NEXT_PUBLIC_FIREBASE_APP_ID\n' +
+    'Firebase features will be disabled.'
+  );
 }
 
-// Export with type assertions - these are only used in client components where they will be defined
-const auth = _auth as Auth;
-const db = _db as Firestore;
-const googleProvider = _googleProvider as GoogleAuthProvider;
+// Safe getters with runtime validation
+export function getAuth(): Auth {
```

Biome (2.4.4)

[error] Shouldn't redeclare 'getAuth'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)